Validation/RecoMuon/plugins/NewMuonTrackValidatorBase.h

```diff
@@ -0,0 +1,281 @@
+#ifndef NewMuonTrackValidatorBase_h
```

Comment:
this is a separate issue I think. the base class serves no purpose except to make things slower/complex